# SOME DESCRIPTIVE TITLE.
# Copyright (C) YEAR THE PACKAGE'S COPYRIGHT HOLDER
# This file is distributed under the same license as the PACKAGE package.
#
# Translators:
# Bart Couvreur <bercie23@yahoo.com>, 2005.
#   <meyskens@me.com>, 2012.
# Miroslav Suchý <msuchy@redhat.com>, 2011.
# Peter van Egdom <p.van.egdom@gmail.com>, 2003-2007.
# Petit Gregory <thunderbirds@pandora.be>, 2003.
# Bart Couvreur <bercie23@yahoo.com>, 2005
# Jaap Schouwenburg <jaapschouwenburg.rh@xs4all.nl>, 2014
# Maarten Eyskens <meyskens@me.com>, 2012
# Miroslav Suchý <msuchy@redhat.com>, 2011
# Peter van Egdom <p.van.egdom@gmail.com>, 2003-2007
# Petit Gregory <thunderbirds@pandora.be>, 2003
# Jan Dobes <jdobes@redhat.com>, 2016. #zanata
# Jiri Dostal <jdostal@redhat.com>, 2018. #zanata
msgid ""
msgstr ""
"Project-Id-Version: Spacewalk\n"
"Report-Msgid-Bugs-To: \n"
"POT-Creation-Date: 2024-04-08 15:54+0200\n"
"PO-Revision-Date: 2018-03-16 11:03+0000\n"
"Last-Translator: Copied by Zanata <copied-by-zanata@zanata.org>\n"
"Language-Team: Dutch (http://www.transifex.com/projects/p/spacewalk/language/"
"nl/)\n"
"Language: nl\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"
"Plural-Forms: nplurals=2; plural=(n != 1);\n"
"X-Generator: Zanata 4.6.2\n"

#: ../src/up2date_client/capabilities.py:94
#, python-format
msgid ""
"This client requires the server to support %s, which the current server does "
"not support"
msgstr ""
"Deze client vereist dat de server %s ondersteunt, de huidige server "
"ondersteunt dit niet"

#: ../src/up2date_client/config.py:162
#, python-format
msgid "%s was not found"
msgstr "%s werd niet gevonden"

#: ../src/up2date_client/rpcServer.py:56
msgid "Error: Server Unavailable. Please try later."
msgstr "Fout: Server niet beschikbaar. Probeer het later opnieuw."

#: ../src/up2date_client/rpcServer.py:182
#, fuzzy
msgid "ERROR: can not find server CA file"
msgstr "FOUT: kan RHNS CA bestand niet vinden"

#: ../src/up2date_client/rpcServer.py:220
msgid "Connection aborted by the user"
msgstr "Verbinding verbroken door de gebruiker"

#: ../src/up2date_client/rpcServer.py:268
msgid "Server has refused connection due to high load"
msgstr "De server weigerde de verbinding omdat er te veel dataverkeer is"

#: ../src/up2date_client/up2dateErrors.py:47
#: ../src/up2date_client/up2dateErrors.py:80
#, python-format
msgid "class %s has no attribute '%s'"
msgstr "klasse %s heeft geen attribuut '%s'"

#: ../src/up2date_client/up2dateErrors.py:107
msgid "RPM error.  The message was:\n"
msgstr "RPM fout. De foutmelding was:\n"

#: ../src/up2date_client/up2dateErrors.py:114
msgid "Password error. The message was:\n"
msgstr "Wachtwoord fout. De foutmelding was:\n"

#: ../src/up2date_client/up2dateErrors.py:118
msgid "RPM dependency error. The message was:\n"
msgstr "RPM afhankelijkheidsprobleem. De foutmelding was:\n"

#: ../src/up2date_client/up2dateErrors.py:128
msgid "Error communicating with server. The message was:\n"
msgstr ""
"Er trad een fout op tijdens het communiceren met de server. De foutmelding "
"was:\n"

#: ../src/up2date_client/up2dateErrors.py:135
msgid "File Not Found: \n"
msgstr "Bestand niet gevonden: \n"

#: ../src/up2date_client/up2dateErrors.py:142
msgid "Delay error from server.  The message was:\n"
msgstr "Delay fout van de server. De foutmelding was:\n"

#: ../src/up2date_client/up2dateErrors.py:178
msgid "Error validating data at server:\n"
msgstr "Fout tijdens het valideren van de data op de server:\n"

#: ../src/up2date_client/up2dateErrors.py:191
msgid ""
"\n"
"Red Hat Network Classic is not supported.\n"
"    "
msgstr ""

#: ../src/up2date_client/up2dateErrors.py:197
msgid "The installation number is invalid"
msgstr "Het installatienummer is ongeldig"

#: ../src/up2date_client/up2dateErrors.py:200
msgid "Error parsing the oemInfo file at field:\n"
msgstr "Fout tijdens het verwerken van het oemInfo bestand bij het veld:\n"

#: ../src/up2date_client/up2dateErrors.py:227
msgid "Network error: "
msgstr "Netwerkfout:"

#: ../src/up2date_client/up2dateErrors.py:295
msgid ""
"\n"
"    Your organization does not have enough Management entitlements to "
"register this\n"
"    system. Please notify your organization administrator of this error.\n"
"    You should be able to register this system after your organization frees "
"existing\n"
"    or purchases additional entitlements. Additional entitlements may be "
"purchased by your\n"
"    organization administrator in the SUSE Customer Center.\n"
"\n"
"    A common cause of this error code is due to having mistakenly setup an\n"
"    Activation Key which is set as the universal default.  If an activation "
"key is set\n"
"    on the account as a universal default, you can disable this key and "
"retry to avoid\n"
"    requiring a Management entitlement."
msgstr ""

#, fuzzy
#~ msgid "Warning: unable to enable rhnsd with systemd"
#~ msgstr "Waarschuwing: kon rhnsd niet opstarten met chkconfig"

#~ msgid "Warning: unable to enable rhnsd with chkconfig"
#~ msgstr "Waarschuwing: kon rhnsd niet opstarten met chkconfig"
